centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/Widgets/centreon-widget-topbottom/src/Label.tsx

```diff
@@ -0,0 +1,46 @@
+import { inc } from 'ramda';
+import { Link } from 'react-router';
+
+import { Tooltip } from '@centreon/ui/components';
+import { Typography } from '@mui/material';
+import { getResourcesUrlForMetricsWidgets } from '../../utils';
+
+import { forwardRef } from 'react';
+import { useTopBottomStyles } from './TopBottom.styles';
+import { Resource } from './models';
+
+interface Props {
+  metricTop: Resource;
+  index: number;
+}
+
+const Label = forwardRef<HTMLParagraphElement, Props>(
+  ({ metricTop, index }, ref) => {
+    const { classes } = useTopBottomStyles({});
+
+    return (
+      <div className={classes.tooltipContainer}>
+        <Tooltip
+          followCursor={false}
+          label={`${metricTop.parentName}_${metricTop.name}`}
+          placement="top"
+        >
+          <Typography className={classes.resourceLabel} ref={ref}>
+            <Link
+              className={classes.linkToResourcesStatus}
+              data-testid={`link to ${metricTop?.name}`}
+              target="_blank"
+              to={getResourcesUrlForMetricsWidgets(metricTop)}
+            >
+              <strong>
+                #{inc(index)} {`${metricTop.parentName}_${metricTop.name}`}
```

Comment:
i think it more clear and easy to read if you do this
{`#${inc(index)} ${metricTop.parentName}_${metricTop.name}`}
it also help konwing that there is a space between

or better to write it in seperate variable

Reply:
Yep write it into a separated variable

Reply:
ok i go with the second option then